public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ConfigEditor.tsx

```diff
@@ -182,7 +159,7 @@ export class ConfigEditor extends PureComponent<Props, State> {
 
     return this.backendSrv
       .datasourceRequest({
-        url: this.state.config.url + workspaceListUrl,
+        url: this.props.options.url + workspaceListUrl,
         method: 'GET',
       })
       .then((result: any) => {
```

Comment:
You could clean up this code the same way you did in the `loadSubscriptions` function